src/AppInstallerCLICore/Commands/UpgradeCommand.cpp

```diff
@@ -252,7 +252,9 @@ namespace AppInstaller::CLI
                 context << SelectLatestApplicableUpdate(true);
             }
 
-            context << InstallSinglePackage;
+            context <<
+                CheckForUnsupportedArgs <<

```

Comment:
Why is this not just inside `InstallSinglePackage`?

Reply:
Moved to `InstallSinglePackage`